api_server/swagger/firecracker.yaml

```diff
@@ -353,6 +353,31 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /vm-config:
+    put:
+      summary: Creates a vm config.
```

Comment:
Both the summary and the description should explicitly mention that this also _starts_ the microVM.